packages/docs/docs/guides/5-request-client/1-payment-networks.md

```diff
@@ -5,42 +5,54 @@ keywords: Request Client, Payment networks
+* Call the `transferWithReference` method of the Ethereum proxy smart contract with the payment address and the reference as parameters
 
-If you opt for the `pn-erc20-proxy-contract` payment channel, the flow for your users will be:
+## ERC20
 
-1. The user allows the smart contract to transfer funds from his address, up to a certain amount you choose.
+ERC20 tokens are based on Ethereum and therefore also use Ethereum addresses. Though we provide an address based payment network. 
+
+### Address-based payment network
+The payment channel `pn-erc20-address-based` can be used for one-shot address that will only receive payments for a single request.
+
+Before using this payment channel, you then have to make sure you create two addresses: one for the payment (or payments if the request is paid in several parts), and one for the refund.
 
-2. He creates the request
+### Proxy-contract payment network
+This is the most convenient way for most use cases. This method is similar to using the proxy contract with `pn-ethereum-input-data`. Note that the smart contract deployed for ERC20 tokens is different than the one deployed for ether.
 
-3. Out of the request creation, you calculate the payment reference with `last8Bytes(hash(lowercase(requestId + salt + address)))`. **TODO: advise on what to do with the payment reference: store, share etc.**
-4. The payer sends a transaction to the proxy contract with the payment reference.
-5. The request balance is updated and if the amount is sufficient, its status turns to 'Paid'.
+## Declarative requests
 
-# REMOVE: Reference from previous docs
-Detection made programatically:
-Types.PAYMENT_NETWORK_ID.BITCOIN_ADDRESS_BASED ('pn-bitcoin-address-based'): handle Bitcoin payments associated to a BTC address to the request. Every transaction hitting this address will be consider as a payment. Eventually, the payer can provide a BTC address for the refunds. Note that the addresses must be used for one and only one request otherwise one transaction will be considered as a payment for more than one request. (see Bitcoin)
-Types.PAYMENT_NETWORK_ID.TESTNET_BITCOIN_ADDRESS_BASED ('pn-testnet-bitcoin-address-based'): Same as previous but for the bitcoin testnet. (for test purpose) (see Bitcoin)
-Types.PAYMENT_NETWORK_ID.ERC20_ADDRESS_BASED ('pn-erc20-address-based'): handle ERC20 payments to an unique Ethereum address associated with the request. Every transaction hitting this address will be consider as a payment. Eventually, the payer can provide an unique Ethereum address for the refunds. Note that the addresses must be used for one and only one request otherwise one transaction will be considered as a payment for more than one request. (see ERC20)
+A request using the declarative payment network allows the payee and the payer to declare the payments and refunds they create and receive.
+In such a request the balance is computed only regarding the declaration made by these two stakeholders.
```

Comment:
it's actually only the payee's declaration that matters for the balance

Reply:
This is not true @benjlevesque. A refund received will be declared by the payer!

Reply:
oh yeah right 😁